src/main/webapp/placeGuides/PlaceGuideOnList.js

```diff
@@ -36,6 +37,44 @@ class PlaceGuideOnList {
     PlaceGuideOnList.close(placeGuideId);
   }
 
+  static expand(placeGuideId) {
+    const divId = 'placeGuideOnList-' + '{' + placeGuideId + '}';
+    const placeGuideDiv = document.getElementById(divId);
+    placeGuideDiv.querySelectorAll('.folded-placeGuide')[0]
+        .style.display = 'none';
+    placeGuideDiv.querySelectorAll('.card-placeGuide')[0]
+        .style.display = 'block';
+    placeGuideDiv.style.padding = '0px';
+  }
+
+  static getBlobSrc(blobKey) {
+    const src = new URL('/serve-blob', document.URL);
+    src.searchParams.append('blob-key', blobKey);
+    return src;
+  }
+
+  static generateQueryString(placeGuideProperties) {
+    const esc = encodeURIComponent;
+    const query = Object.keys(placeGuideProperties)
+        .map(function(k) {
+          return esc(k) + '=' + esc(placeGuideProperties[k]);
```

Comment:
Wondering if there is a nice URL api from Javascript to add parameters more cleanly?

Reply:
There is, "url.searchParams.append(name, value)", but that would not align with the string parsing method written by Dennis. I would rather leave it as he did it before in this case.